physionet-django/physionet/settings/base.py

```diff
@@ -48,6 +48,7 @@
     'ckeditor',
     # 'django_cron',
     'background_task',
+    'session_security',
```

Comment:
Let's make this configurable (same as SSO and Lightwave) - an env variable should control whether this is even enabled.